core/flyout.js

```diff
@@ -1162,6 +1162,10 @@ Blockly.Flyout.prototype.createBlockFunc_ = function(originBlock) {
       return;
     }
     Blockly.Events.disable();
+    // Disable workspace resizing.  Reenable at the end of the drag. We do
+    // not need the workspace to resize in the middle of place new block
```

Comment:
How about
"Disable workspace resizing.  Reenable at the end of the drag.  This avoids a spurious resize between creating the new block and placing it in the workspace."